loop.py

```diff
@@ -59,6 +60,41 @@ def compute_unfairness(A, R):
     return np.abs(np.asarray(A) - np.asarray(R))
 
 
+def compute_max_unfairness(A, R, k):
+    min_value = sys.maxsize
+    unfairness = (np.asarray(A) - np.asarray(R))[::-1]
```

Comment:
?

Reply:
My idea was to go through the list with unfairness per item in reverse order, as to prioritize those size - k items, since they are at the end of the spectrum in terms of unfairness

Reply:
The list is not sorted?

Reply:
But it **does** represent the rankings right? So the item at index 0 is the highest ranked item and the one at k the lowest.

Reply:
> So the item at index 0 is the highest ranked item and the one at k the lowest.
..........The list is not sorted